common/src/test/scala/io/phdata/pulse/common/SolrServiceTest.scala

```diff
@@ -167,17 +166,17 @@ class SolrServiceTest extends FunSuite with BaseSolrCloudTest {
     val collectionName = TestUtil.randomIdentifier()
     solrService.createCollection(collectionName, 1, 1, "testconf", null)
 
-    val properties = Map("hostname" -> "host1.com")
-
-    val document1 = DocumentConversion.toSolrDocument(
-      new LogEvent(None,
-                   "ERROR",
-                   "1970-01-01T00:00:00Z",
-                   "ERROR",
-                   "message",
-                   "thread oxb",
-                   Some("Exception in thread main"),
-                   Some(properties)))
+    val document1 = DocumentConversion.mapToSolrDocument(
+      Map(
+        "id"         -> None,
+        "category"   -> "ERROR",
+        "timestamp"  -> "1970-01-01T00:00:00Z",
+        "level"      -> "ERROR",
+        "message"    -> "message",
+        "threadName" -> "thread oxb",
+        "throwable"  -> "Exception in thread main",
+        "hostname"   -> "host1.com"
+      ).asInstanceOf[Map[String, String]])
```

Comment:
Cast?

Reply:
Without casting `mapToSolrDocument` complains about the `id=None`.

Reply:
Think you meant to respond to this with:

 "Can you share that error? Solr should be creating its own ids, and I believe that endpoint takes a SolrDocument type, so this is odd."

The error is with `mapToSolrDocument` and it is a `Type Mismatch: Map[String, Serializable] does not conform to type Map[String, String]`.

I will just remove the `"id" -> None`.  I think the way the code works, a `None` would not make it to `mapToSolrDocument` and that was an oversight by me.